elearning/overview/intro.rst

```diff
@@ -0,0 +1,55 @@
+=========
+Elearning
```

Comment:
You have two separate files with the same title. Given that this one will loose its meaning as soon as you add new pages about eLearning, I would rename it to something more definitive, and the filename too !